packages/react-reconciler/src/ReactFiberScheduler.js

```diff
@@ -1054,6 +1151,16 @@ function renderRoot(
 
   const expirationTime = root.nextExpirationTimeToWorkOn;
 
+  let prevInteractions: Set<Interaction> | null = null;
+  if (enableInteractionTracking) {
+    // We're about to start new tracked work.
+    // Restore pending interactions so cascading work triggered during the render phase will be accounted for.
+    prevInteractions = ((__interactionsRef: any): InteractionsRef).current;
```

Comment:
All these any casts are too fragile. Let’s make sure that we cast this at the export type in one place.